[inductor] Add type hints to functions in mkldnn_fusion.py

aakhundov opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #131820

Summary: ATT

Test Plan: lintrunner

Reviewers:

Subscribers:

Tasks:

Tags:

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

aakhundov committed:
[inductor] Add type hints to functions in mkldnn_fusion.py

Summary: ATT

Test Plan: lintrunner

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

aakhundov commented:
@pytorchbot merge

aakhundov commented:
@pytorchbot rebase

aakhundov committed:
Update

[ghstack-poisoned]

aakhundov commented:
@pytorchbot merge

aakhundov commented:
@pytorchbot merge

aakhundov committed:
Update on "[inductor] Add type hints to functions in mkldnn_fusion.py"


Summary: ATT

Test Plan: lintrunner

Reviewers:

Subscribers:

Tasks:

Tags:

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire chauhang

[ghstack-poisoned]

aakhundov commented:
@pytorchbot merge

aakhundov commented:
@pytorchbot merge -f "skip the stuck [inductor-periodic / cuda12.1-py3.10-gcc9-sm80](https://github.com/pytorch/pytorch/actions/runs/10105654459/job/27947429029) job as irrelevant for the type-hinting PR in the MKLDNN fusion passes"

clee2000 commented:
@pytorchbot revert -m "reverting this to revert something else, only action you should need to do is to rebase and merge again, sorry for the churn" -c ghfirst

aakhundov commented:
@pytorchbot rebase

aakhundov committed:
Update on "[inductor] Add type hints to functions in mkldnn_fusion.py"


Summary: ATT

Test Plan: lintrunner

Reviewers:

Subscribers:

Tasks:

Tags:

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire chauhang

[ghstack-poisoned]

aakhundov committed:
Update on "[inductor] Add type hints to functions in mkldnn_fusion.py"


Summary: ATT

Test Plan: lintrunner

Reviewers:

Subscribers:

Tasks:

Tags:

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire chauhang

[ghstack-poisoned]

aakhundov commented:
@pytorchbot merge

ZainRizvi commented:
@pytorchbot revert -c ghfirst -m "Sorry, had to revert this to revert another PR that depends on this change"

aakhundov commented:
@pytorchbot rebase
